database/buildSchema/29_review_response.sql

```diff
@@ -22,22 +22,21 @@ CREATE TABLE public.review_response (
 	status public.review_response_status default 'Draft'
 );
 
--- set review response original_response_id (the response that links to application id should be available for all reaponses)
--- also flatten out review response chain by providing template_element_id in review_response
+-- set review response original_review_response_id (the response that links to application id should be available for all reaponses)
```

Comment:
You already changed this on the earlier PR, so maybe merge that in here now?

Reply:
i think it's in there now (if you look at full diff)